content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -312,6 +322,62 @@ The value of the label is the namespace name.
 While NetworkPolicy cannot target a namespace by its name with some object field, you can use the
 standardized label to target a specific namespace.
 
+## Pod lifecycle
+
+We say that a NetworkPolicy is applied to a selected pod, when isolation rules are enforced
+for that pod. At that point pod can never establish an outbound connection or receive
+an inbound connection that is not allowed by the NetworkPolicy. Allow rules will be applied
+eventually after the isolation rules (that includes applying both at the same time).
+That means, that once the NetworkPolicy is applied to a pod, the pod should be able to handle situations,
+when it can't establish a connection that is allowed by an existing NetworkPolicy for some time.
+The requirements for Allow rules are less strict, because a set of allowed endpoints may be constantly
+changing if the rules are based on a `namespaceSelector` or a `podSelector`.
+
+Every NetworkPolicy will be applied to all selected pods eventually, but there is no
+way to tell from the existing Kubernetes API when exactly the network policy is applied to a specific
+pod. The only to check if the network policy is applied to a pod, is to run a connectivity check.
```

Comment:
```suggestion
pod. The only way to check if the network policy is applied to a pod, is to run a connectivity check.
```

Even that connectivity check isn't 100% reliable, IMO.

Reply:
I was trying to specify at least 1 way to check if the pod is isolated or not. I can just remove it, or replace with some better option? @danwinship do we want to answer the question "how I am supposed to figure out if the network policy is applied to a pod or not"?